java/monkey/src/main/java/root/ast/LetStatement.java

```diff
@@ -0,0 +1,18 @@
+package root.ast;
+
+import root.Token;
+
+public class LetStatement extends Statement {
```

Comment:
Where possible, can we  maybe go into Packages with the base abstract on the top level of `ast`. Just makes it harder to read and mentally organize when you're dealing with 3-4th level `Abstract`

Reply:
I will also tack on an apologizes on this suggestion for @xNaCly , I know we had talked about avoiding nesting too many packages, but it was the only way I could think of to help with mentally with the nesting of all these abstractions. It lets me see them grouped in my IDE way better, which helps keep them "grouped" in my head at least.
Also follows normal Packaging behavior from my understanding.

Reply:
What about this structure?
<img width="246" alt="image" src="https://github.com/ThePrimeagen/ts-rust-zig-deez/assets/39140806/1ac00d85-1273-47bc-ae93-1164663788e6">

Reply:
That Structure is better..